tools/clang/test/HLSLFileCheck/hlsl/intrinsics/power/pow2.hlsl

```diff
@@ -0,0 +1,12 @@
+// RUN: %dxc -T ps_6_0 -E main %s | FileCheck %s
+
+// Make sure pow 2 not generate log -> mul 2 -> exp.
+// Log
+// CHECK-NOT:call float @dx.op.unary.f32(i32 23,
+// CHECK-NOT:fmul fast float %{{.*}}, 2.000000e+00
+// Exp
+// CHECK-NOT:call float @dx.op.unary.f32(i32 21,
```

Comment:
Shouldn't this check *for* something, to ensure it at least compiled successfully, and ideally, resolved to some mul of value by same value?
Such as:
```
// CHECK: fmul fast float %[[a:.*]], %[[a]]
// CHECK: fmul fast float %[[b:.*]], %[[b]]
// CHECK: fmul fast float %[[c:.*]], %[[c]]
// CHECK: fmul fast float %[[d:.*]], %[[d]]
```